composer/utils/parallelism.py

```diff
@@ -85,7 +89,7 @@ class FSDP2Config:
     #       in most of our use cases, we can decouple these two attributes from the FSDP2Config class.
     activation_checkpointing: bool = False
     activation_cpu_offload: bool = False
-    state_dict_type: str = 'sharded'
+    state_dict_type: str = 'full'
```

Comment:
I guess this can be either `sharded` or `full` but I thought it might be best to keep this equivalent to what we have in `FSDP1` (which is `full`)...